cpp/ps0001/p301F solving problem 1.cpp

```diff
@@ -4,21 +4,22 @@ using namespace std;
 
 int main()
 {
-    int num1;
-    int num2;
-    int num3;
-
-
-cout<<"enter the first number /n1" << endl;
-cin>> num1;
-
-cout<<"enter the second number /n1" << endl;
-cin>>num2;
-
-
-cout<<"enter the third number /n1" << endl;
-cin>> num3;
-
-    cout << num1+num2+num3 << endl;
+    // prompting users for 3 numbers to add 1/2
+    short num1;
+    short num2;
+    short num3;
+    
+    cout << "enter the first number" << endl;
+    cin >> num1;
+    
+    cout << "enter the second number" << endl;
+    cin >> num2;
+    
+    cout << "enter the third number" << endl;
+    cin >> num3;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add